docs/core/tools/global-json.md

```diff
@@ -14,7 +14,9 @@ ms.assetid: 96102f96-d403-4385-8ef6-5d80e406eb0c
 
 # global.json reference
 
-The global.json file allows selection of the .NET Core tools version being used through the `sdk` property. 
+The global.json file allows selection of the .NET Core tools version being used through the `sdk` property.
+
+.NET Core CLI tools look for this file in the current working directory (which does not have to be the project directory), or one of its parent directories.
```

Comment:
*Nice add!* :rocket:

Small suggestions ...
* If you want to update it, the "global.json" should be in italics.
* If you want to update it, the JSON file should have two-space indents.
* Use contraction: "does not" :point_right: "doesn't"
* There's a comma splice there ... a few ways to fix it: Drop the parentheses from the phrase and place a comma before *which* (forming a dependent clause), or you could leave the parentheses and remove the comma before *or* (there are two objects of the preposition *in* there ... so no comma with the conjunction).